pkg/store/azure_keyvault_store.go

```
@@ -0,0 +1,222 @@
+func (s *KeyVaultStore) Delete(stack string, component string, key string) error {
+	secretName, err := s.getKey(stack, component, key)
+	if err != nil {
+		return err
+	}
+
+	_, err = s.client.DeleteSecret(context.Background(), secretName, nil)
+	if err != nil {
+		var respErr *azcore.ResponseError
+		if errors.As(err, &respErr) {
+			switch respErr.StatusCode {
+			case 404:
```

golangci-lint (1.64.8)

[error] add-constant: avoid magic numbers like '404', create a named constant for it

(revive)